node/src/main/scala/coop/rchain/node/diagnostics/effects/package.scala

```diff
@@ -163,6 +161,13 @@ package object effects {
   def metrics[F[_]: Sync]: Metrics[F] =
     new Metrics[F] {
       import kamon._
+      import kamon.trace.{Span => KSpan}
+
+      case class KamonSpan(span: KSpan) extends Span[F] {
+        @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+        override def mark(name: String): F[Unit] = Sync[F].delay { span.mark(name) }
+        override def close(): F[Unit]            = Sync[F].delay { span.finish() }
```

Comment:
Why the rename from `finish` to `close`?

Reply:
seemed more aligned with our naming scheme